### DISCUSSION TITLE
Monarch Money Icon

### DISCUSSION THREAD

**Discussion by ZacSted** [elrumo/macOS_Big_Sur_icons_replacements#1499] [Icon request]
![Monarch Money](https://github.com/user-attachments/assets/6f18e748-486e-4fc0-9f73-8d4c72bde1b6)

**Comment by bunkerdesign** [comment]
![Monarch Money](https://github.com/user-attachments/assets/f0479ffe-37de-4d9f-be3f-db0a67189b6d)
